auth-api/src/auth_api/services/membership.py

```diff
@@ -289,11 +290,11 @@ def deactivate_membership(self, **kwargs):
 
         # if this is a member removing another member, check that they admin or owner
         if self._model.user.username != user_from_context.user_name:
-            check_auth(org_id=self._model.org_id, one_of_roles=(COORDINATOR, ADMIN))
+            check_auth(org_id=self._model.org_id, one_of_roles=(COORDINATOR, ADMIN, STAFF))
```

Comment:
As discussed, the check_auth call updates should be able to function as is without these changes, since we are basing off of the users membership type